_i18n/en.yml

```diff
@@ -559,7 +559,7 @@ faq:
   aasicresistance2: >
     Monero fixes this problem by being ASIC-resistant: it uses an algorithm (@randomx) that strongly reduces the efficiency of ASICs, making them not profitable to build. Miners can use common consumer hardware, which allows them to compete fairly. The Monero network is currently protected by thousands of miners using 'regular' computers. This results in a network much harder to attack, no miner having significant advantage over other miners (they all use more or less the same hardware).
   qmaxsupply: What is Monero's maximum supply?
-  amaxsupply: Monero has a fixed emission rate, not a set maximum supply. Around May 2022, Monero's emission will drop to and permanently remain at 0.3 XMR per minute (0.6 XMR per @block). This is approximately 1% inflation for the first year and will approach 0% inflation in future years. This @tail-emission allows for permanent incentives to secure Monero, even in the far future, while keeping inflation at a very low percent.
+  amaxsupply: Monero has a fixed emission rate, not a set maximum supply. On November 8th 2024, Monero's emission has started and permanently will remain at 0.3 XMR per minute (0.6 XMR per @block). This is approximately 1% inflation for the first year and will approach 0% inflation in future years. This @tail-emission allows for permanent incentives to secure Monero, even in the far future, while keeping inflation at a very low percent.
```

Comment:
The date here is off

Reply:
yep, https://p2pool.io/tail.html  2022-06-09 / 9th June 2022